codestodians/food-basket-front-end
# Add set, delete and get cookie functions

#### What does this PR do?
Adds helpers functions to set, get and delete cookies
#### Description of Task to be completed?
Create Functions for getting and setting cookies
#### How should this be manually tested?
- Check out to the `ft-cookie-helpers-151293458` branch.
- Import and make a call to `getCookie`, `setCookie` and `deleteCookie` functions from `helpers.js`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#151293458](https://www.pivotaltracker.com/n/projects/2091604/stories/151293458)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
